cl/api/management/commands/cl_retry_webhooks.py

```diff
@@ -51,6 +53,29 @@ def retry_webhook_events() -> int:
     return len(webhook_events_to_retry)
 
 
+def delete_old_webhook_events() -> int | None:
+    """Delete webhook events older than DAYS_TO_DELETE days.
+    This is executed once a day at 12:00 UTC (4:00 PDT)
+
+    :return: The number of deleted webhooks events or None if it's not time to
+    execute the method.
+    """
+
+    minute_of_the_day = now().hour * 60 + now().minute
+    # 12:00 UTC -> 4:00 PDT (minute 720 of the day)
+    if minute_of_the_day == 720:
+        # Older than DAYS_TO_DELETE days
+        days = DAYS_TO_DELETE
+        older_than = now() - timedelta(days=days)
+        webhooks_events_to_delete = WebhookEvent.objects.filter(
+            date_created__lt=older_than
+        )
+        deleted_count = len(webhooks_events_to_delete)
+        webhooks_events_to_delete.delete()
+        return deleted_count
```

Comment:
We shouldn't do `len()` unless we want the records, so this would be better:

```suggestion
        deleted_count = webhooks_events_to_delete.count()
        return deleted_count
```

But, when you do a delete from Django (and SQL, I think), you get the number of affected rows back, so, you don't even need to do `count()`. 

So it's something like:

```suggestion
        return webhooks_events_to_delete.delete()
```

(Though I think Django returns a tuple when you do a delete, so we'll have to deal with that, but otherwise, I think that's the best way to do it.)

Reply:
Yes, delete() returns a tuple of (int, dict) with the total of objects deleted, a dict specifying the deleted objects if more than one type of object was deleted in the process.

In this case, I used the total of objects deleted.